bimmer_connected/vehicle/remote_services.py

```diff
@@ -4,17 +4,26 @@
 import datetime
 import json
 import logging
-from typing import TYPE_CHECKING, Dict, Optional, Union
+from typing import TYPE_CHECKING, Any, Dict, Optional, Union
 
 from bimmer_connected.api.client import MyBMWClient
 from bimmer_connected.const import (
     REMOTE_SERVICE_POSITION_URL,
     REMOTE_SERVICE_STATUS_URL,
     REMOTE_SERVICE_URL,
+    VEHICLE_CHARGING_PROFILE_SET_URL,
+    VEHICLE_CHARGING_SETTINGS_SET_URL,
+    VEHICLE_CHARGING_START_STOP_URL,
     VEHICLE_POI_URL,
 )
-from bimmer_connected.models import PointOfInterest, StrEnum
+from bimmer_connected.models import ChargingSettings, PointOfInterest, StrEnum
```

Comment:
Add the function to check the validity of your json string:

`from bimmer_connected.utils import is_json`